lib/between_meals/util.rb

```diff
@@ -90,6 +92,26 @@ def port_open?(port)
       end
       return false
     end
+
+    def chef_zero_running?(port, use_ssl)
+      Timeout.timeout(1) do
+        begin
+          uri = URI("http://localhost:#{port}")
```

Comment:
It seems, since http may be a lie (it may be https), and you have no additional URL bits, you can simplify this a lot

```
http = Net::HTTP.new('localhost', port)
if use_ssl
  http.use_ssl = true
  http.verify_mode = OpenSSL::SSL::VERIFY_NONE
end
res = http.get('/')
return res['Server']
```

Reply:
Yup, that's way cleaner

Reply:
Still need to return res['Server'] == 'chef-zero', however.